src/client/core/utils/dom.js

```diff
@@ -390,8 +390,16 @@ export function isTopWindow (win) {
     }
 }
 
-export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
-    const iframes = (iframeDestinationWindow || window).document.getElementsByTagName('iframe');
+export function findIframeByWindow (iframeWindow) {
+    const iframes = [];
+
+    document.querySelectorAll('*').forEach(elem => {
```

Comment:
You should use the native methods from the `hammerhead.nativeMethods`  for querySelector, querySelectorAll functions.

Consider using the [find](https://github.com/DevExpress/testcafe-hammerhead/blob/a878aefb01e45c3dc58d9d92da6e618cbc494c4c/src/client/utils/dom.ts#L261) method from hammerhead here.